user_docs/en/userGuide.t2t

```diff
@@ -1757,7 +1757,7 @@ Until you do, a status of "remove" will be shown for that add-on in the list.
 
 To disable an add-on, press the disable button.
 To enable a previously disabled add-on, press the enable button.
-You can disable an add-on if the add-on status indicates it is running or enabled, or enable it if the add-on is suspended or disabled.
+You can disable an add-on if the add-on status indicates it is  enabled, or enable it if the add-on is disabled.

```

Comment:
For consistency with other lines in the user guide perhaps use quotes around enabled / disabled. Elsewhere it is seems to say "a status of". I don't think that this needs to be consistent with that.

Also could you add a note that when all addons are disabled via "restart with addons disabled" the status will show "enabled after restart"